.github/workflows/config.yml

```diff
@@ -4,6 +4,24 @@ on: [push]
 
 jobs:
 
+  snyk_scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.14'
+      - uses: snyk/actions/setup@master
+      - uses: actions/checkout@v2
```

Comment:
Are these `uses` arrays defining when the scan is going to run?

Reply:
I believe uses just means that these steps need to run before any actions